src/Moryx.AbstractionLayer.TestTools/InMemoryUnitOfWorkFactoryBuilder.cs

```diff
@@ -39,12 +55,13 @@ public static class InMemoryUnitOfWorkFactoryBuilder
         /// <typeparam name="T"></typeparam>
         /// <param name="factory"></param>
         /// <returns></returns>
-        public static UnitOfWorkFactory<T> EnsureDbIsCreated<T>(this UnitOfWorkFactory<T> factory)
```

Comment:
Unfortunately, we publish the Moryx.AbstractionLayer.TestTools project as a package, i.e. this should be a breaking change if I'm not mistaken 🙈